shell/platform/android/android_surface_software.cc

```diff
@@ -108,7 +139,11 @@ bool AndroidSurfaceSoftware::OnScreenSurfaceResize(const SkISize& size) const {
 bool AndroidSurfaceSoftware::SetNativeWindow(
     ftl::RefPtr<AndroidNativeWindow> window,
     PlatformView::SurfaceConfig config) {
-  return true;
+  native_window_ = std::move(window);
+  int32_t window_format = ANativeWindow_getFormat(native_window_->handle());
+  if (window_format >= 0)
+    GetSkColorType(window_format, &target_color_type_);
```

Comment:
If this returns false (when the window format it unknown), we want the validity check to fail.